.changeset/lemon-singers-sparkle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-openhim': minor
```

Comment:
@hunterachieng i am not sure if it should be a minor, these are breaking changes. Are these functions use in prod ? or this is a new adaptor ?

Reply:
@mtuchi  It is a new adaptor. All those functions were implemented in the new version

Reply:
Ooh okay thats fine then 👍🏽

Reply:
This is not a new adaptor and these are breaking changes.

Any workflows which use, for example, `registerClient()` will break

Reply:
@josephjclark  I have updated it to a major changeset